presto-druid/src/main/java/com/facebook/presto/druid/DruidAggregationProjectConverter.java

```diff
@@ -111,7 +112,7 @@ private DruidExpression handleDateTruncationViaDateTruncation(
         }
 
         inputColumn = timeConversion.getArguments().get(0).accept(this, context).getDefinition();
-        inputTimeZone = timeConversion.getArguments().size() > 1 ? getStringFromConstant(timeConversion.getArguments().get(1)) : DateTimeZone.UTC.getID();
+        inputTimeZone = timeConversion.getArguments().size() > 1 ? getStringFromConstant(timeConversion.getArguments().get(1)) : TimeZone.getTimeZone(ZoneOffset.UTC).getID();
```

Comment:
I've verified both the old and new code return the string "UTC" here. Arguably I should simply use that string literal for this case.